moha/hamiltonians.py

```diff
@@ -703,3 +707,170 @@ def __init__(self,
             J_ax=J_ax,
             connectivity=connectivity
         )
+
+
+class HamTJUV(HamPPP, HamHeisenberg):
+    r"""t-J-U-V Hamiltonian."""
+
+    def __init__(self,
+                 connectivity: Union[list, np.ndarray],
+                 alpha=-0.414,
+                 beta=-0.0533,
+                 u_onsite=None,
+                 gamma=None,
+                 charges=None,
+                 sym=1,
+                 atom_dictionary=None,
+                 bond_dictionary=None,
+                 orbital_overlap=None,
+                 affinity_dct=None,
+                 Rxy_list=None,
+                 J_eq=None,
+                 J_ax=None):
+        r"""
+        Initialize t-J-U-V Hamiltonian.
+
+        Parameters
+        ----------
+        connectivity_ppp: Union[list, np.ndarray]
```

Comment:
we need to make sure that docstring consistent with the parameters. For example, there is no parameter connectivity_ppp in the input parameters.